release/models/interfaces/openconfig-interfaces.yang

```diff
@@ -720,21 +727,25 @@ module openconfig-interfaces {
       type boolean;
       description
         "When set to true, the interface is a dedicated
-        management interface that is not connected to dataplane
-        interfaces.  It may be used to connect the system to an
-        out-of-band management network, for example.";
+        management interface for external connectivity to the system.
+        This interface is not connected to an INTEGRATED_CIRCUIT and
+        therefore not typically used for dataplane traffic.  It may be
```

Comment:
this is the only part of the description that seems slightly off to me.  I'm not sure I can come up with a better description, but why the use of capitalised 'INTEGRATED_CIRCUIT'?  I think I'm  likely missing the significance of this term - it makes sense that a dedicated management only interface is not designed to make use of the devices dataplane, but what is the relevance of not connecting to an IC? Surely there could be a separate IC specifically designed to handle management plane traffic?

Reply:
Yes, I think this is a good question -- if this port is not connected to some forwarding ASIC, what is it connected to? It seems like it /could/ be connected to an ASIC, just there is no guarantee that it is, since it might just be a NIC-like local Ethernet port on the supervisor/routing-engine/CPM/....

(Simon, I think the relevance of the `INTEGRATED_CIRCUIT` being uppercase is that this is the identity in the `openconfig-platform` model for ASICs/NPUs.)